components/Snackbar/tests/unit/MDCSnackbarMessageViewTests.m

```diff
@@ -91,4 +101,32 @@ - (void)testAccessibilityHintSetFromSnackbarMessageProperty {
                         self.message.accessibilityHint);
 }
 
+- (void)testSnackbarSetAccessibiltyViewIsModalForActionSnacbars {
+  // Given
+  self.manager.internalManager.isVoiceOverRunningOverride = YES;
+  MDCSnackbarMessageAction *action = [[MDCSnackbarMessageAction alloc] init];
+  action.title = @"Tap Me";
+  self.message.action = action;
+  self.manager.shouldSetAccessibilityViewIsModal = YES;
+
+  // When
+  [self.manager showMessage:self.message];
+  [NSRunLoop.mainRunLoop runMode:NSDefaultRunLoopMode beforeDate:[NSDate distantFuture]];
+
+  // Then
+  XCTAssertTrue(self.manager.internalManager.overlayView.accessibilityViewIsModal);
+}
+
+- (void)testSnackbarAccessibiltyViewIsModalShouldBeNo {
+  // Given
+  self.manager.shouldSetAccessibilityViewIsModal = YES;
+
+  // When
+  [self.manager showMessage:self.message];
+
+  // Then
+  XCTAssertFalse(self.manager.internalManager.overlayView.accessibilityViewIsModal);
+}
+
+
```

Comment:
Suggestion: we may want a test that confirms that `shouldSetAccessibilityViewIsModal` set to NO results in `accessibilityViewIsModal` being NO as well.